### PR TITLE
added recipe for iso8601

### PR DESCRIPTION
**Pull request by ChrisBarker-NOAA**
built fine for me with py2.7 and py3.5

### CODE REVIEW

File: recipes/iso8601/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,37 @@
+{% set version = "0.1.11" %}
+{%set sha256 = "e8fb52f78880ae063336c94eb5b87b181e6a0cc33a6c008511bac9a6e980ef30" %}
```
Comment (jakirkham) [inline recipes/iso8601/meta.yaml]: nit: `{%set` -> `{% set`

**Comment by jakirkham** [comment]
One small nit. Otherwise LGTM.

File: recipes/iso8601/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+about:
+    home: https://bitbucket.org/micktwomey/pyiso8601
+    license: MIT
+    summary: 'Simple module to parse ISO 8601 dates'
+    doc_url: http://pyiso8601.readthedocs.io/en/latest/
+
+extra:
+    recipe-maintainers:
+        - ChrisBarker-NOAA
+        - JamesMakela-NOAA
```
Comment (jakirkham) [inline recipes/iso8601/meta.yaml]: @JamesMakela-NOAA, is it ok with you to be added as a maintainer of `iso8601`?

Comment (ChrisBarker-NOAA) [inline recipes/iso8601/meta.yaml]: @JamesMakela-NOAA <https://github.com/JamesMakela-NOAA>, is it ok with you
to be added as a maintainer of iso8601?

James is on my team, and introduced this dependency to our stack. And he's
on vacation, so if we don't hear from him soon, please push anyway.

-CHB

Comment (JamesMakela-NOAA) [inline recipes/iso8601/meta.yaml]: @jakirkham: Yes, it will be fine to add me as a maintainer.

Comment (jakirkham) [inline recipes/iso8601/meta.yaml]: Thanks for confirming.

**Comment by jakirkham** [comment]
@micktwomey, just letting you know we have added `iso8601` to [conda-forge]( http://conda-forge.github.io/ ). This will allow users to easily install this with [`conda`]( http://conda.pydata.org/docs/ ) (`conda install -c conda-forge iso8601`). The maintenance burden here is light as all testing releases are built and deployed automatically with CI. Changes to how it is being built can be controlled by changing the recipe in [this repo]( https://github.com/conda-forge/iso8601-feedstock ). If you have any interest in being added as a maintainer (to make new releases for instance), we would love to have you on board. Thanks for your work on `iso8601`. 😄

**Pull request merged**